x/wasm/types/types.go

```diff
@@ -429,3 +429,18 @@ func (a AccessConfig) AllAuthorizedAddresses() []string {
 	}
 	return []string{}
 }
+
+type TxContracts map[string]struct{}
```

Comment:
Can we make this map private such that we can change it any time later? Maybe as a private field of `TxContracts`